templates/css/example.css

```diff
@@ -95,6 +95,24 @@ code {
 .abe-code-preview {
   background: #f9f7f7;
 }
+.abe-story-preview {
```

Comment:
does it make sense to have css for stories in a separate file and add it to the page only if it's a story instead for all the pages?

Reply:
I wouldn't that this is justified as long as it's only 5 extra lines